plugin-manifest.json

```diff
@@ -14,7 +14,7 @@
   "platform": "android",
   "author_name": "Alexander Mykhailiutenko",
   "author_email": "amykh@scand.com",
-  "manifest_version": "0.0.1",
+  "manifest_version": "0.0.2",
```

Comment:
You don't need to update the manifest if you are just increasing the patch version (0.0.x) as the Android dependency version is with a + (0.0.+).

Reply:
I usually do it to keep manifest updated according to committed version.